### PR TITLE
fixed ul, li, indentation

### PR DESCRIPTION
**Pull request by 81jumpstr**

### CODE REVIEW

File: page-upskill.php [hunk 1]
```diff
@@ -7,37 +7,41 @@
...
+      <div class="thumbnails">
+        <?php the_post_thumbnail('thumbnail'); ?>
+      </div>
+    <?php endif; ?>
+
+          <small>
+            <?php the_category(); ?>
+          </small>
+
+          <li>
```
Comment (royscheeren) [inline page-upskill.php]: li in wrong place

**Review by royscheeren — CHANGES_REQUESTED** [review]

**Comment by 81jumpstr** [comment]
Thinking about it for too long. My best says; This seems to make more sense to me.

**Comment by royscheeren** [comment]
Sorry dude but no. I might not be explaining it correctly, it doesn't seem to stick.

**Comment by 81jumpstr** [comment]
Attempt #3

**Comment by 81jumpstr** [comment]
Page-upskill-markup, placing ul…steps :

-	Prepare to put main content in a div or container
-	Check whether posts are available
-	Be ready to post when found
-	Retrieve and be ready to post relevant content and page
-	Retrieve be ready to post the available article
-	Retrieve and be ready to include the post title
-	Check whether thumbnails available
-	Be ready to post thumbnails in assigned div
-	Stop checking for thumbnails
-	Retrieve and be ready to print out the post category
-	Create or open an unordered list
-	Post all retrieved content queueing to be posted
-	Once all above steps have been executed, stop looping to find available posts
-	Then quit checking whether posts are available
-	Close the main content container

**Review by royscheeren — COMMENTED** [review]
Besides indentation the formatting is looking good. But, if I look at the file name I see it's a page. There will be only one post ever in the loop.

**Comment by 81jumpstr** [comment]
I woke up this morning remembering that you did mention at some point that I don't need to create a list. Just couldn't remember in regard to what. I was about to trace back our conversations...so my brain is working...ok, cool

**Comment by 81jumpstr** [comment]
I know we said we don't need a list for page template files. 
I just want to see if I fully understand, should I come across a similar task in the future.

File: page-upskill.php [hunk 2]
```diff
@@ -7,36 +7,35 @@
...
-            <?php the_content(); ?>
-          </li>
-        </ul>
-      <?php
-          get_template_part( 'content', 'page' );
-      ?>
-          <article id="post-<?php the_ID(); ?>" <?php post_class(); ?>>
-
-          <?php the_title('<h1 class="entry-title">','</h1>' ); ?>
+            <?php the_title('<h1 class="entry-title">','</h1>' ); ?>
```
Comment (royscheeren) [inline page-upskill.php:16]: Your title is a part of the article, so it should go in there

File: page-upskill.php [hunk 3]
```diff
@@ -7,36 +7,35 @@
...
+            <article <?php post_class(); ?>>
+            <?php get_template_part( 'content', 'page' ); ?>
```
Comment (royscheeren) [inline page-upskill.php]: What's in your template part?

File: page-upskill.php [hunk 4]
```diff
@@ -7,36 +7,35 @@
...
+            <small>
+              <?php the_category(); ?>
+            </small>
+
+            <?php the_content(); ?>
+          </li>
+      <?php
+        endwhile;
+      ?>
+        </ul>
```
Comment (royscheeren) [inline page-upskill.php]: You only need a list if you display multiple posts, which the main loop on a page won't have.

There's two parts to this whole story:
- Formatting your html and php correctly
- Knowing when to use which elements

**Review by royscheeren — CHANGES_REQUESTED** [review]

File: page-upskill.php (hunk 2 above)
Comment (81jumpstr) [reply page-upskill.php:16]: So, this part is good?

File: page-upskill.php (hunk 3 above)
Comment (81jumpstr) [reply page-upskill.php]: Should I just remove it? Another 1 of those "in the future perhaps" things.

File: page-upskill.php (hunk 4 above)
Comment (81jumpstr) [reply page-upskill.php]: Will remove ul and li's

File: page-upskill.php [hunk 5]
```diff
@@ -9,10 +9,12 @@
...
   <div class="main-section">
     <?php 
-      if ( have_posts() ) : ?>           
+      if ( have_posts() ) : ?>
+      	<ul class="posts">           
       <?php 
         while ( have_posts() ) : the_post(); 
       ?>
+          <li>  
             <?php the_title('<h1 class="entry-title">','</h1>' ); ?>
```
Comment (royscheeren) [inline page-upskill.php:16]: Your title is still not part of your article.

**Review by royscheeren — CHANGES_REQUESTED** [review]

File: page-upskill.php (hunk 5 above)
Comment (81jumpstr) [reply page-upskill.php:16]: You meant inside the article tag?

**Pull request closed without merging**